webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/checks/TriggerPolicyUpdate.scala

```diff
@@ -65,10 +67,11 @@ class TriggerPolicyUpdate(
     try {
       if (file.exists) {
```

Comment:
Why keep that test? I would just remove it and deprecate both flag files. And document it. 
And in 7.0 remove the files enterily.

Reply:
so that people are not surprised as some are used to use it

Reply:
ok, so make the test only to adapt message, and put out the start, wichi is not subject to condition. 
And I think we need to adapt the message to tell "you don't need that file, since 6.1.x start is automatic". 
And I think we must do it only in 6.1: 6.0 is EOL and it complicate the explanation about when the behavior changed.

Reply:
but I do indeed need to remove the other one